sklearn/compose/_target.py

```diff
@@ -22,14 +22,28 @@ class TransformedTargetRegressor(BaseEstimator, RegressorMixin):
     QuantileTransformer or as a function and its inverse such as ``log`` and
     ``exp``.
 
-    The computation during ``fit`` is::
```

Comment:
I think simply adding a blank line after this line can solve the problem.

Reply:
You're right, adding the blank line totally works! Thanks for the tip, I was only aware of the `.. code-block::` syntax for code examples.